src/schema_registry/telemetry/metrics.py

```diff
@@ -0,0 +1,27 @@
+"""
+Copyright (c) 2024 Aiven Ltd
+See LICENSE for details
+"""
+
+from opentelemetry.metrics import Counter, Histogram, UpDownCounter
+from schema_registry.telemetry.meter import Meter
+from typing import Final
+
+
+class Metrics:
```

Comment:
```suggestion
class HttpRequestMetrics:
...
    def new_request(self): # increment in progress and also the total requests.
    def finish_request(self, duration: float): # reduce in progress, record request duration.
```
What do you think?

Reply:
I like the idea, provide future scoping of metrics 👍